build-all.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
this is very specific to building the email app, so in any case move it to the freenet-email-app directory; 
those apps will be eventually move dout of this repo

Reply:
yes I would love. at this moment example apps rely on module folder. I do not have better implementation idea.

Reply:
nothing stops you from moving this to:
`apps/freenet-email-app`, just use relative paths if necessary